### PR TITLE
Update CHANGELOG.md for the 2.3.1 release

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +24,12 @@
 * Fix custom key notation not restored correctly after restart [#4136](https://github.com/mixxxdj/mixxx/pull/4136)
 * Traktor S3: Disable scratch when switching decks to prevent locked scratch issue [#4073](https://github.com/mixxxdj/mixxx/pull/4073)
 * FFmpeg: Ignore inaudible samples before start of stream [#4245](https://github.com/mixxxdj/mixxx/pull/4245)
+* Controller Preferences: Don't automatically enable checkbox if controller is disabled [#4244](https://github.com/mixxxdj/mixxx/pull/4244) [lp:1941042](https://bugs.launchpad.net/mixxx/+bug/1941042)
+* Tooltips: Always show tooltips in preferences [#4198](https://github.com/mixxxdj/mixxx/pull/4198) [lp:1840493](https://bugs.launchpad.net/mixxx/+bug/1840493)
+* Tooltips: Use item label for tooltips in library side bar. Show id when debugging. [#4247](https://github.com/mixxxdj/mixxx/pull/4247)
```
Comment (Holzhaus) [inline CHANGELOG.md:29]:
```suggestion
* Tooltips: Use item label for tooltips in library side bar and show ID when debugging. [#4247](https://github.com/mixxxdj/mixxx/pull/4247)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +24,12 @@
...
+* Library sidebar: Also activate items on PageUp/Down events. [#4237](https://github.com/mixxxdj/mixxx/pull/4237)
+* Fix handling of preview button cell events in developers mode. [#4264](https://github.com/mixxxdj/mixxx/pull/4264) [lp:1929141](https://bugs.launchpad.net/mixxx/+bug/1929141)
```
Comment (Holzhaus) [inline CHANGELOG.md:31]:
```suggestion
* Fix handling of preview button cell events in developer mode. [#4264](https://github.com/mixxxdj/mixxx/pull/4264) [lp:1929141](https://bugs.launchpad.net/mixxx/+bug/1929141)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,6 +38,9 @@
 * .desktop file has be renamed to org.mixxx.Mixxx.desktop according to Freedesktop standards [#4206](https://github.com/mixxxdj/mixxx/pull/4206)
 * Uses system provided hidapi library if version >= 0.10.1 [#4215](https://github.com/mixxxdj/mixxx/pull/4215)
 * Please update PortAudio to [19.7](https://github.com/PortAudio/portaudio/releases/tag/v19.7.0) if you have not done so already. This is required for Mixxx to work with PipeWire via the JACK API for many devices.
+* We now install multiple sizes of rasterized icons [#4204](https://github.com/mixxxdj/mixxx/pull/4204)
```
Comment (Holzhaus) [inline CHANGELOG.md:41]:
```suggestion
* Install multiple sizes of rasterized icons [#4204](https://github.com/mixxxdj/mixxx/pull/4204)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -30,6 +38,9 @@
...
+* Fixd detection of SoundTouch pkgconfig file and version [#4209](https://github.com/mixxxdj/mixxx/pull/4209)
```
Comment (Holzhaus) [inline CHANGELOG.md:42]:
```suggestion
* CMake: Fixed detection of SoundTouch pkgconfig file and version [#4209](https://github.com/mixxxdj/mixxx/pull/4209)
```

**Review by Holzhaus — COMMENTED** [review]
Thanks. Please rerun pre-commit and squash after applying changes.

**Comment by daschuer** [comment]
Done

**Comment by daschuer** [comment]
Done

**Review by Be-ing — APPROVED** [review]

**Pull request merged**